sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -399,6 +418,43 @@ def test_global_fanout(self):
           | beam.CombineGlobally(combine.MeanCombineFn()).with_fanout(11))
       assert_that(result, equal_to([49.5]))
 
+  def test_combining_with_accumulation_mode(self):
+    # PCollection will contain elements from 1 to 5.
+    elements = [i for i in range(1, 6)]
+
+    ts = TestStream().advance_watermark_to(0)
+    for i in elements:
+      ts.add_elements([i])
+    ts.advance_watermark_to_infinity()
+
+    options = PipelineOptions()
+    options.view_as(StandardOptions).streaming = True
```

Comment:
Note that due to https://issues.apache.org/jira/browse/BEAM-8601 this makes it impossible to use as a validates runner test.

Reply:
We will avoid making it validates runner.